resources/linux/debian/templates.template

```diff
@@ -0,0 +1,7 @@
+Template: @@NAME@@/add-microsoft-repo
+Type: boolean
+Default: true
+Description: Add Microsoft repository for Visual Studio Code?
+ The installer determined that you should most likely install add the
```

Comment:
Pinging @ntrogh for thoughts on the copy here. I like the question on the first line, but I wonder if the longer description can contain just an explanation, such as:

The installer would like to add the Microsoft repository to update packages such as code through apt.

Reply:
https://github.com/microsoft/vscode/pull/218463/files#diff-579de8e37ffbe1385f33eb1a401b261b8af44a0ad9074e75e726161a55955441R4 could work as well

Reply:
I went with a mix of the two. If you have objections, let me know

Reply:
@rzhao271 @m-byte the latest version of the text looks good to me. The longer description is precise and clear about the intent.